### ISSUE TITLE
Still not in package management

### ISSUE THREAD

**Issue by jult** [aperezdc/ngx-fancyindex#108]
Is there a particular reason NGINX does not include this module by default,
[despite pointing to it for years on end](https://www.nginx.com/resources/wiki/modules/fancy_index/) as if it does ?
I would have understood if it had just been created, but it's now been there for over 8 years.
Come on.

Pretty sad I have to resort to installing from sources on all my servers just for this package alone, whereby the config line changes at every new NGINX release, which can be disastrously time-
consuming (and needless too, since it has been done before!).

In other words;
Anyone here knows of a debian repo that maintains a stable secure nginx version *with* this module included ?

For the life of me I don't understand why linux society keeps making it so hard for itself
to run a secure up-to-date webserver, while *all* sources and defaults are widely available
and very well known. Such a waste of valuable time!

**Comment by alehaa** [comment]
Debian has this [package](https://packages.debian.org/buster/libnginx-mod-http-fancyindex) included. Just install `nginx-full`.

**Comment by jult** [comment]
Oh wow, thanks. That was embarrassing. It's even there in sid. https://packages.debian.org/sid/libnginx-mod-http-fancyindex
